fdbserver/BlobManager.actor.cpp

```diff
@@ -5061,6 +5104,17 @@ ACTOR Future<Void> purgeRange(Reference<BlobManagerData> self, KeyRangeRef range
 		}
 	}
 
+	// no longer needed, can save memory
+	activeRanges.clear();
+
+	TraceEvent("PurgeGranulesTraversalGotLatest", self->id)
```

Comment:
@sfc-gh-jslocum can you please confirm if this trace is not spammy? Thanks

Reply:
There are several other SevInfo traces in this function so it's not a concern